integration/tests/epochs/suite.go

```diff
@@ -667,35 +688,34 @@ func (s *Suite) assertNetworkHealthyAfterANChange(ctx context.Context, env templ
 // this func can be used to perform sanity.
 // 1. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
 func (s *Suite) assertNetworkHealthyAfterVNChange(ctx context.Context, _ templates.Environment, rootSnapshot *inmem.Snapshot, _ *StakedNodeOperationInfo) {
-	bootstrapHead, err := rootSnapshot.Head()
-	require.NoError(s.T(), err)
-
-	header, err := s.client.GetLatestSealedBlockHeader(ctx)
-	require.NoError(s.T(), err)
-
-	// head should now be at-least 20 blocks higher from when we started
-	require.True(s.T(), header.Height-bootstrapHead.Height >= 20, fmt.Sprintf("expected head.Height %d to be higher than head from the snapshot the node was bootstraped with bootstrapHead.Height %d.", header.Height, bootstrapHead.Height))
+	// assert atleast 20 blocks have been finalized since the node replacement
+	s.assertLatestFinalizedBlockHeightHigher(ctx, rootSnapshot, uint64(20))
 }
 
 // assertNetworkHealthyAfterLNChange after an collection node is removed or added to the network
 // this func can be used to perform sanity.
 // 1. Submit transaction to network that will target the newly staked LN by making sure the reference block ID
-// is after the first epoch starts.
-// 2. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
+// is after the first epoch.
 func (s *Suite) assertNetworkHealthyAfterLNChange(ctx context.Context, _ templates.Environment, rootSnapshot *inmem.Snapshot, _ *StakedNodeOperationInfo) {
-	fullAccountKey := sdk.NewAccountKey().
-		SetPublicKey(unittest.PrivateKeyFixture(crypto.ECDSAP256, crypto.KeyGenSeedMinLenECDSAP256).PublicKey()).
-		SetHashAlgo(sdkcrypto.SHA2_256).
-		SetWeight(sdk.AccountKeyWeightThreshold)
+	// At this point we have reached epoch 1 and our new LN node should be the only LN node in the network.
+	// To validate the LN joined the network successfully and is processing transactions we submit a
+	// create account transaction and assert there are no errors.
+	s.submitSmokeTestTransaction(ctx)
+}
 
+// assertNetworkHealthyAfterSNChange after replacing a consensus node in the test and waiting until
+// the epoch transition we should observe blocks finalizing and we should be able to submit a transaction
+// that will indicate overall network health
+// 1. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
+// 2. Submit transaction to network
+func (s *Suite) assertNetworkHealthyAfterSNChange(ctx context.Context, _ templates.Environment, rootSnapshot *inmem.Snapshot, _ *StakedNodeOperationInfo) {
+	// assert atleast 20 blocks have been finalized since the node replacement
+	s.assertLatestFinalizedBlockHeightHigher(ctx, rootSnapshot, uint64(20))
```

Comment:
I think this is mostly re-treading assertions which are made earlier in the test, in the [generic test function](https://github.com/onflow/flow-go/blob/169120222281f7071c3c92416a43ae3f669be847/integration/tests/epochs/epoch_test.go#L193-L194). Since the `EpochSetup` phase is much longer than 20 views, it is very likely that this is actually checking that some view _within the first epoch_ has been finalized (ie. before the consensus node swap at the epoch boundary). 

In the generic test function before this is called, we ensure that several views into the second epoch are sealed, which gives us confidence that finalization and sealing continues after the consensus node swap -- ie. the new consensus node must be participating in finalization and sealing, because there are only 2 nodes. That is really the condition we want to check for the SN test. Perhaps we can replace the `assertLatestFinalizedBlockHeightHigher` call with some comments explaining how the assertion in the generic test function provides evidence of the new consensus node contributing to finalization/sealing.

Reply:
https://github.com/onflow/flow-go/pull/1856/commits/151e4aecad9d9365e0c5888625ebcddc303c8448